lib/URI.pm

```diff
@@ -5,16 +5,22 @@ use warnings;
 
 our $VERSION = '5.11';
 
+# 1=version 5.10 and earlier; 0=version 5.11 and later
+use constant HAS_RESERVED_SQUARE_BRACKETS => $ENV{URI_HAS_RESERVED_SQUARE_BRACKETS} ? 1 : 0;
+
 our ($ABS_REMOTE_LEADING_DOTS, $ABS_ALLOW_RELATIVE_SCHEME, $DEFAULT_QUERY_FORM_DELIMITER);
+our $RESERVED_SQUARE_BRACKETS = $ENV{URI_RESERVED_SQUARE_BRACKETS} || 0;  # 1=version 5.10 and earlier; 0=version 5.11 and later
```

Comment:
This variable doesn't appear to be used.

Reply:
Thank you. That's a left over from a previous try. Will be removed.